### DISCUSSION TITLE
+7 phone code 2fa

### DISCUSSION THREAD

**Discussion by Corax73** [community/community#136327] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

You suggest enabling 2fa, I can only use SMS, but for my phone there is no +7 code in your drop-down menu. Please provide this option for the dialing code +7.

**Comment by vimukthijayasanka** [comment]
It sounds like you're encountering a limitation where the +7 dialing code (used for Russia and Kazakhstan) is missing from the drop-down menu when trying to enable SMS-based two-factor authentication (2FA). This is likely a bug or oversight in the application's user interface.

To resolve this issue, here are some steps you can take:

Check for Updates: Ensure that you're using the latest version of the application. Sometimes, bugs like this can be resolved in newer updates.

Report the Issue: If updating doesn't solve the problem, you should report it to the support team or through the application's official issue tracker (e.g., GitHub if it's an open-source project). Include details about the issue, such as the missing +7 dialing code, and any other relevant information (e.g., your operating system and app version).

Alternative 2FA Methods: If the issue persists, consider using another method for 2FA, such as an authentication app (Google Authenticator, Authy, etc.), which doesn't rely on SMS codes and can be more secure.

Workaround: If possible, try selecting another country code that might be a typo or similar to +7. While this isn't ideal, it might be a temporary workaround until the issue is resolved.

Reporting the issue is the most important step, as it will alert the developers to the missing dialing code and help them address the problem in a future update.

**Comment by ebndev** [comment]
Hey there @Corax73,

Thanks so much for reaching out in GitHub's Community! If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

We put together a [FAQ](https://github.com/orgs/community/discussions/68240) in case you have any additional questions that may be answered there!

**Reply by SabitMussabek** [reply]
Why isn’t Kazakhstan on the list?
